packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -194,6 +194,51 @@ class _AnimatedState extends State<AnimatedWidget> {
 /// animated by a [CurvedAnimation] set to [Curves.elasticIn]:
 /// {@animation 300 378 https://flutter.github.io/assets-for-api-docs/assets/widgets/slide_transition.mp4}
 ///
+/// {@tool snippet --template=stateful_widget_scaffold_center_freeform_state}
```

Comment:
Would have loved to use the new `dartpad` tool, but the code throws a "Script error" in there. (works fine on device).

/cc @RedBrogdon @ferhatb

Reply:
Hrmm. This is an interesting challenge. DartPad pulls code from the API doc server (currently the master version server), and therefore can't load the sample until the PR lands and kicks off a new doc roll.

Add to that the difficulty that we won't be able to run bleeding-edge master on the dartpad backend without being at least a little bit behind. It's too much to ask the backend to stop and rebuild flutter anytime a commit lands, at least right now.

When we were designing the initial integration of this feature, I was mostly concerned with converting widgets and samples already in the SDK. You're already the second person to ask for a better solution for fresh code, though, so I'll need to find a way to provide it.